appveyor.yml

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Yep! You can see them [here](https://www.appveyor.com/docs/build-environment/#java) and I've added some extra steps to ensure it's on 1.8.0 (commit pending).